docs/contributor/06-10-certs.md

```diff
@@ -1,14 +1,19 @@
 # Certification management
 
-![Certification management diagram](../assets/certs.svg)
-
-Certification reconciliation is triggered by one of the three events: scheduled reconciliation, editing [BtpOperator CR](../../api/v1alpha1/btpoperator_types.go), or using custom watchers founded on Secret and Webhook resources.
+BTP Manager maintains two Secrets, `ca-server-cert` and `webhook-server-cert`, which are used to allow for communication within BTP Operator webhooks and thus allow for the creation of resources like ServiceInstances and ServiceBindings. The `ca-server-cert`, `webhook-server-cert`, and their CA Bundles are kept in sync by using the reconciliation mechanism, which means that every manual change in these resources the user makes automatically triggers the regeneration of all three resources.
```

Comment:
Can we switch to more active voice?

```suggestion
BTP Manager maintains two Secrets, `ca-server-cert` and `webhook-server-cert`. They are used for communication within BTP Operator webhooks and for the creation of resources like ServiceInstances and ServiceBindings. The [reconciliation mechanism](#reconciliation_mechanism) syncs the two Secrets and their CA Bundles, which means that whenever the user manually changes them, they are automatically regenerated.
```

Which are "all three"? The two Secrets and their CA Bundles? Aren't those four at least ("bundles" implies at least two)?